config/config.ts

```diff
@@ -120,31 +120,6 @@ const config: Configuration = {
                                 branch: 'master',
                             },
                         },
-                        {
-                            name: 'Experimental',
-                            key: 'experimental',
-                            url: 'https://flybywirecdn.com/addons/a32nx/experimental',
```

Comment:
Add these URLs to line 108 subsequent to transition current users of the experimental version to dev